src/components/Menu.js

```diff
@@ -9,37 +9,32 @@ const StyleMenu = styled.ul`
 
     @media (min-width: 768px) {
```

Comment:
You can take values from Atomic layout breakpoints so responsive is consistent:

```diff
- @media (min-width: 768px) {
+ @media (min-width: ${Layout.breakpoints.md.minWidth}) {
```

To use `Layout` you would need to import it first:

```js
import Layout from 'atomic-layout'
```